.env

```diff
@@ -1,3 +1,5 @@
 export PG_DATABASE_URL=postgres://granite:password@localhost:5432/granite_db
 export MYSQL_DATABASE_URL=mysql://granite:password@localhost:3306/granite_db
```

Comment:
do these lines need to be changed so that they're using the correct variables for travis's containers?

Reply:
No they don't have to change. They will be the same for every database container that is build

Reply:
I cant tell if we miscommunicated or not, so forgive me if not. 

What I mean is this. I believe that the password for Travis's database container is 'password', but surely the username for Travis's database isn't 'granite'?